src/db/migrations/0071_adorable_gabe_jones.sql

```
@@ -0,0 +1,54 @@
+CREATE INDEX "idx_backups_status_core" ON "backups" USING btree ("status","deleted_at");--> statement-breakpoint
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)